### PR TITLE
cleanup Nodejs env var checks examples

### LINKED ISSUE #1359: Cleanup Nodejs env var checks in examples
Same as https://github.com/iotaledger/iota-sdk/issues/1358, but for Nodejs and in many examples there are also checks missing in the first place, so they should be added before (for example process.env.EXPLORER_URL)

### PR DESCRIPTION
**Pull request by shreykul**
# Description of change
Added a list of env for iteration as done in the python cleanup

## Links to any relevant issues
fixes #1359

## Type of change

- Bug fix (a non-breaking change which fixes an issue)
 
## How the change has been tested

Ran unit tests locally.

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by thibault-martinez** [comment]
Hello, thanks for your PR!
Please do the changes to the `develop` branch, `1.1` has been released already and the branch will be removed soon.

File: bindings/nodejs/examples/exchange/1-create-account.ts [hunk 1]
```diff
@@ -12,56 +12,48 @@ require('dotenv').config({ path: '.env' });
...
+        
+       
 
         const wallet = new Wallet(walletOptions);
 
         // Mnemonic only needs to be set the first time.
+        if(process.env.MNEMONIC){
         await wallet.storeMnemonic(process.env.MNEMONIC);
 
+        }
```
Comment (Thoralf-M) [inline bindings/nodejs/examples/exchange/1-create-account.ts]: This extra if everywhere is very annoying, can't we just do this?
```suggestion
        await wallet.storeMnemonic(process.env.MNEMONIC as string);
```

Comment (Thoralf-M) [reply bindings/nodejs/examples/exchange/1-create-account.ts]: Btw to format the examples you can use `yarn run format` in `bindings/nodejs`

Comment (shreykul) [reply bindings/nodejs/examples/exchange/1-create-account.ts]: Thanks for the correction, I'll do that and I checked the yarn run format later after seeing the test failure.

File: bindings/nodejs/examples/how_tos/alias/create.ts [hunk 2]
```diff
@@ -16,17 +16,15 @@ require('dotenv').config({ path: '.env' });
...
-        throw new Error(
-            '.env STRONGHOLD_PASSWORD is undefined, see .env.example',
-        );
-    }
+    for (const envVar of [
+        'FAUCET_URL',
+        'WALLET_DB_PATH',
+        'STRONGHOLD_PASSWORD',
+    ])
+        if (!process.env.FAUCET_URL) {
```
Comment (Thoralf-M) [inline bindings/nodejs/examples/how_tos/alias/create.ts]:
```suggestion
        if (!(envVar in process.env)) {
```

File: bindings/nodejs/examples/exchange/1-create-account.ts [hunk 3]
```diff
@@ -51,7 +42,9 @@ async function run() {
         const wallet = new Wallet(walletOptions);
 
         // Mnemonic only needs to be set the first time.
-        await wallet.storeMnemonic(process.env.MNEMONIC);
+        if (process.env.MNEMONIC) {
+            await wallet.storeMnemonic(process.env.MNEMONIC);
+        }
```
Comment (Thoralf-M) [inline bindings/nodejs/examples/exchange/1-create-account.ts]:
```suggestion
        await wallet.storeMnemonic(process.env.MNEMONIC as string);
```

File: bindings/nodejs/examples/exchange/2-generate-address.ts [hunk 4]
```diff
@@ -12,22 +12,20 @@ require('dotenv').config({ path: '.env' });
...
-        }
-        if (!process.env.STRONGHOLD_PASSWORD) {
-            throw new Error(
-                '.env STRONGHOLD_PASSWORD is undefined, see .env.example',
-            );
-        }
+        for (const envVar of ['WALLET_DB_PATH', 'STRONGHOLD_PASSWORD'])
+            if (!(envVar in process.env)) {
+                throw new Error(
+                    `.env ${envVar} is undefined, see .env.example`,
```
Comment (Thoralf-M) [inline bindings/nodejs/examples/exchange/2-generate-address.ts:18]:
```suggestion
                    `.env ${envVar} is undefined, see .env.example`,
```

File: bindings/nodejs/examples/exchange/5-send-amount.ts [hunk 5]
```diff
@@ -12,25 +12,24 @@ require('dotenv').config({ path: '.env' });
...
-            throw new Error('.env EXPLORER_URL is undefined, see .env.example');
-        }
+        for (const envVar of [
+            'WALLET_DB_PATH',
+            'STRONGHOLD_PASSWORD',
+            'EXPLORER_URL',
+        ])
+            if (!(envVar in process.env)) {
+                throw new Error(
+                    '.env WALLET_DB_PATH is undefined, see .env.example',
```
Comment (Thoralf-M) [inline bindings/nodejs/examples/exchange/5-send-amount.ts]:
```suggestion
                    `.env ${envVar} is undefined, see .env.example`,
```

**Comment by shreykul** [comment]
Sorry it is just so confusing, i changed the files locally but weren't able to see the changes here so yeah that's why it took so much time in an easy thing...

File: bindings/nodejs/examples/how_tos/sign_secp256k1_ecdsa/sign-secp256k1_ecdsa.ts [hunk 6]
```diff
@@ -42,7 +41,8 @@ async function run() {
         // A mnemonic can be generated with `Utils.generateMnemonic()`.
         // Store the mnemonic in the Stronghold snapshot, this needs to be done only the first time.
         // The mnemonic can't be retrieved from the Stronghold file, so make a backup in a secure place!
-        await secretManager.storeMnemonic(process.env.MNEMONIC);
+
```
Comment (Thoralf-M) [inline bindings/nodejs/examples/how_tos/sign_secp256k1_ecdsa/sign-secp256k1_ecdsa.ts]:
```suggestion
```

File: bindings/nodejs/examples/how_tos/sign_and_verify_ed25519/sign-ed25519.ts [hunk 7]
```diff
@@ -48,7 +47,8 @@ async function run() {
         // A mnemonic can be generated with `Utils.generateMnemonic()`.
         // Store the mnemonic in the Stronghold snapshot, this needs to be done only the first time.
         // The mnemonic can't be retrieved from the Stronghold file, so make a backup in a secure place!
-        await secretManager.storeMnemonic(process.env.MNEMONIC);
+
```
Comment (Thoralf-M) [inline bindings/nodejs/examples/how_tos/sign_and_verify_ed25519/sign-ed25519.ts]:
```suggestion
```

File: bindings/nodejs/examples/how_tos/alias/destroy.ts [hunk 8]
```diff
@@ -48,7 +46,10 @@ async function run() {
         );
 
         // To sign a transaction we need to unlock stronghold.
-        await wallet.setStrongholdPassword(process.env.STRONGHOLD_PASSWORD);
+
```
Comment (Thoralf-M) [inline bindings/nodejs/examples/how_tos/alias/destroy.ts]:
```suggestion
```

File: bindings/nodejs/examples/how_tos/alias/create.ts [hunk 9]
```diff
@@ -42,7 +40,10 @@ async function run() {
         console.log(`Aliases BEFORE:\n`, balance.aliases);
 
         // To sign a transaction we need to unlock stronghold.
-        await wallet.setStrongholdPassword(process.env.STRONGHOLD_PASSWORD);
+
```
Comment (Thoralf-M) [inline bindings/nodejs/examples/how_tos/alias/create.ts]:
```suggestion
```

File: bindings/nodejs/examples/how_tos/accounts_and_addresses/create-address.ts [hunk 10]
```diff
@@ -26,7 +23,10 @@ async function run() {
         const account = await wallet.getAccount('Alice');
 
         // To create an address we need to unlock stronghold.
-        await wallet.setStrongholdPassword(process.env.STRONGHOLD_PASSWORD);
+
```
Comment (Thoralf-M) [inline bindings/nodejs/examples/how_tos/accounts_and_addresses/create-address.ts]:
```suggestion
```

File: bindings/nodejs/examples/how_tos/accounts_and_addresses/consolidate-outputs.ts [hunk 11]
```diff
@@ -37,7 +32,10 @@ async function run() {
         const account = await wallet.getAccount('Alice');
 
         // To create an address we need to unlock stronghold.
-        await wallet.setStrongholdPassword(process.env.STRONGHOLD_PASSWORD);
+
```
Comment (Thoralf-M) [inline bindings/nodejs/examples/how_tos/accounts_and_addresses/consolidate-outputs.ts]:
```suggestion
```

File: bindings/nodejs/examples/secret_manager/stronghold.ts [hunk 12]
```diff
@@ -30,7 +27,10 @@ async function run() {
         // A mnemonic can be generated with `Utils.generateMnemonic()`.
         // Store the mnemonic in the Stronghold snapshot, this needs to be done only the first time.
         // The mnemonic can't be retrieved from the Stronghold file, so make a backup in a secure place!
-        await strongholdSecretManager.storeMnemonic(process.env.MNEMONIC);
+
```
Comment (Thoralf-M) [inline bindings/nodejs/examples/secret_manager/stronghold.ts]:
```suggestion
```

**Comment by Thoralf-M** [comment]
No worries, thanks for your PR!

**Review by Thoralf-M — APPROVED** [review]

**Review by Alex6323 — APPROVED** [review]

**Pull request merged**